app/routes/events/EventDetailRoute.js

```diff
@@ -124,11 +125,14 @@ const mapDispatchToProps = {
   isUserFollowing,
   deleteComment
 };
-
+/*
 const loadData = async ({ params: { eventId }, loggedIn }, dispatch) => [
   await dispatch(fetchEvent(eventId)),
   loggedIn && (await dispatch(isUserFollowing(eventId)))
 ];
+*/
+const loadData = async ({ params: { eventId }, loggedIn }, dispatch) =>
+  await dispatchInOrder([fetchEvent, isUserFollowing], dispatch, eventId);
```

Comment:
In this refactoring you are missing the `loggedIn` check before calling `isUserFollowing`. Is that intentional?

Reply:
It is not, nice catch.